### PR TITLE
Remove arrow in relevance sorting label

### PR DESCRIPTION
**Pull request by awead**
Fixes issue reporting in #914 by updating the catalog_controller generator to match the current one in Blacklight.

### CODE REVIEW

**Comment by mjgiarlo** [comment]
Hate to be that guy, but could you add a test that fails w/o this change and passes w/ it?  It seems we haven't had tests for this in the past.

**Comment by awead** [comment]
@mjgiarlo yeah, good point. Fortunately, there was a quick view test I could piggy-back off of.

**Comment by mjgiarlo** [comment]
:+1:

File: spec/views/catalog/sort_and_per_page.html.erb_spec.rb [hunk 1]
```diff
@@ -15,13 +15,18 @@
...
-
-
   it 'appears on page without error' do
     render
     page = Capybara::Node::Simple.new(rendered)
     expect(page).to have_selector('span.page_entries', count: 1)
     expect(rendered).to include("<strong>4</strong> - <strong>6</strong> of <strong>20</strong>")
   end
+  
+  it 'does not use a directional arrow for relevance sorting' do
```
Comment (jcoyne) [inline spec/views/catalog/sort_and_per_page.html.erb_spec.rb]: I prefer we just test for what it does do. There are lots of things it does not do that we are not testing for.

File: spec/views/catalog/sort_and_per_page.html.erb_spec.rb [hunk 2]
```diff
@@ -15,13 +15,18 @@
...
+    render
+    page = Capybara::Node::Simple.new(rendered)
+    expect(rendered).to include("<li><a href=\"/catalog?sort=score+desc%2C+date_uploaded_dtsi+desc\">relevance</a></li>")
```
Comment (jcoyne) [inline spec/views/catalog/sort_and_per_page.html.erb_spec.rb]: This is sufficient, we don't need the next line.

**Pull request merged**